starlette/types.py

```diff
@@ -1,9 +1,18 @@
+import sys
 import typing
 
-Scope = typing.MutableMapping[str, typing.Any]
-Message = typing.MutableMapping[str, typing.Any]
+if sys.version_info >= (3, 10):  # pragma: no cover
+    from typing import TypeAlias
+else:  # pragma: no cover
+    from typing_extensions import TypeAlias
 
-Receive = typing.Callable[[], typing.Awaitable[Message]]
-Send = typing.Callable[[Message], typing.Awaitable[None]]
+Scope: TypeAlias = typing.MutableMapping[str, typing.Any]
+Message: TypeAlias = typing.MutableMapping[str, typing.Any]
 
-ASGIApp = typing.Callable[[Scope, Receive, Send], typing.Awaitable[None]]
+Receive: TypeAlias = typing.Callable[[], typing.Awaitable[Message]]
+Send: TypeAlias = typing.Callable[[Message], typing.Awaitable[None]]
+
+ASGIApp: TypeAlias = typing.Callable[[Scope, Receive, Send], typing.Awaitable[None]]
```

Comment:
Can we change the order back to what it was?

Reply:
Looking at https://github.com/encode/starlette/blob/master/starlette/types.py the order is still the same

Reply:
Oh, sorry. hahaha